Gemfile

```diff
@@ -27,4 +29,4 @@ gem "chromedriver-helper", "1.0.0"
 # https://github.com/teamcapybara/capybara/blob/master/History.md#version-2141
 # So until we understand what the problem is and how to resolve it we need to
 # lock the project to a version we know works.
-gem "capybara", "2.14.0"
+gem "capybara", "3.18.0"
```

Comment:
If we can confirm the javascript prompt using this new version then again the need to lock it is no longer needed so you can drop it altogether.